### DISCUSSION TITLE
Ultima Template: The top menu bar utilities and ui-kit menus do not navigate when clicked on in the Ultima template.

### DISCUSSION THREAD

**Discussion by nexgen-ui** [primefaces/community#2747] [PrimeVue Templates] [answered]
Select the UI Kit or Utilites menu from the top bar in the Ultima Template.
Select a menu item from the sub-menu.
Nothing happens

**Comment by tugcekucukoglu** [comment, marked as answer]
I'll fix them for the next version.

Thank you!
